esmvaltool/recipes/recipe_autoassess_radiation_rms_Amon_all.yml

```diff
@@ -1,10 +1,18 @@
+# ESMValTool
+# recipe_autoassess_radiation_rms_Amon_all.yml
 ---
 documentation:
-  description: Diagnostics of spatial RMS errors of radiation and clouds.
+  description: |
+    Diagnostics of spatial RMS errors of radiation and clouds.
+
   authors:
     - tsus_yo
     - read_si
```

Comment:
you can safely remove Simon and Yoko, they've left the group (Yoko is still at the MetOffice but out the ESMValTool loop and Simon is left climate science)

Reply:
If they contributed to the recipe they can stay as authors, even if they left the group.
Only the maintainer needs to the still there (and active).

Reply:
aha! in that case keep Simon, can remove Yoko still since I rewrote everything she did (unfortunately); @alistairsellar has contributed to the non-V1 Autoassess recipes (stratosphere, landsurface_*) via assistance for me so pls include him